packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```
@@ -316,63 +316,61 @@ const maxDate = (a: number | string, b: string) => {
+                const currentMin =
+                    typeof acc[0] === 'string' ? parseFloat(acc[0]) : acc[0];
+                const currentMax =
+                    typeof acc[1] === 'string' ? parseFloat(acc[1]) : acc[1];
+
+                if (!isNaN(currentNumber)) {
+                    const min =
+                        currentNumber < currentMin ? currentNumber : currentMin;
+                    const max =
+                        currentNumber > currentMax ? currentNumber : currentMax;
 
                     return [min, max];
-                } else if (
-                    typeof value === 'string' ||
-                    typeof value === 'number'
-                ) {
-                    // is number or numeric string
-                    const currentNumber =
-                        typeof value === 'string' ? parseFloat(value) : value;
-                    const currentMin =
-                        typeof acc[0] === 'string'
-                            ? parseFloat(acc[0])
-                            : acc[0];
-                    const currentMax =
-                        typeof acc[1] === 'string'
-                            ? parseFloat(acc[1])
-                            : acc[1];
-
-                    if (!isNaN(currentNumber)) {
-                        const min =
-                            currentNumber < currentMin
-                                ? currentNumber
-                                : currentMin;
-                        const max =
-                            currentNumber > currentMax
-                                ? currentNumber
-                                : currentMax;
-
-                        return [min, max];
-                    }
                 }
-                return acc;
-            },
-            [0, 0],
-        );
+            } else {
+                // TODO: this case comes up more than it should given that
```

should we add a log here instead of silently fail ?

---

Would you log to the console? I wasn't entirely sure what it would say. You think it should log to the console? I put this here cause I figure it's helpful for devs, but I didn't throw or anything because there is nothing a user can do about it.

---

I'm going to merge without this. But I might look into this TODO. If you want, I'm happy to add a log here.